README.md

```diff
@@ -6,12 +6,26 @@ It allows you to upload a release to PLTcloud as part of your CI workflow.
 
 ## Input variables
 
-You must provide:
+Mandatory:
 
-  - `API_TOKEN`: PLTcloud API token.
-  - `PROJECT_UUID`: Project id for PLTcloud project.
-  - `FILES`: List of files for upload. Files can be specified with patterns such as: `**/prefix*`, `grandparent/**/child?`, `**/parent/*`, or even just `**`
-  - `VERSION`: Release version
+| input          | description                      |
+| -------------- | -------------------------------- |
+| `API_TOKEN`    | PLTcloud API token.              |
+| `PROJECT_UUID` | Project id for PLTcloud project. |
+| `FILES`        | List of files for upload. Files can be specified with patterns such as: `**/prefix*`, `grandparent/**/child?`, `**/parent/*`, or even just `**` |
+| `VERSION`      | Release version. (default: `${{ github.ref }}`) |
+
+Optional:
+
+| input              | description                       |
+| ------------------ | --------------------------------- |
+| `VERBOSE`          | Verbose output (default: `false`) |
+| `DEPLOYMENT_GROUP_UUID` | Deployment Group UUID |
+| `AUTO_DEPLOY`      | Auto deploy release to PLT (default: `false`) |
+| `CHECK_NAME`       | GitHub check name (default: `PLTcloud` |
```

Comment:
I think we should probably leave the last for as undocumented  since they are internal implementation details for Hardware CI. Deployment group uuid and auto deploy are definitely neededfor hardware ci, https://docs.pltcloud.com/pltcloud/hardware-ci/ .

Reply:
It also seems that when setting CHECK_NAME is set to anything else than "PLTcloud", PLT Hardware CI stops working, because of https://github.com/bcdevices/pltcloud-ci/blob/2e2631629644fdd47bb1490b857d1b12942e7bf9/internal/gh/client.go#L87